### PR TITLE
{bio}[foss/2016b] BLAST+ 2.6.0

### PR DESCRIPTION
**Pull request by SethosII**
This adds the current `BLAST+` for the `foss-2016b` toolchain.

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca018.camp.thecrick.org - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4d51510abfa15359c0f4db54d7035854 for a full test report.

File: easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+homepage = 'http://blast.ncbi.nlm.nih.gov/'
+description = """Basic Local Alignment Search Tool, or BLAST, is an algorithm
+ for comparing primary biological sequence information, such as the amino-acid
+ sequences of different proteins or the nucleotides of DNA sequences."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++14'}
+
+sources = ['ncbi-blast-%(version)s+-src.tar.gz']
+source_urls = ['http://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/%(version)s/']
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BLAST+/BLAST+-2.6.0-foss-2016b-Python-2.7.12.eb]: Please add an SHA256 checksum

**Comment by SethosII** [comment]
@verdurin I added the requested changes.

**Comment by verdurin** [comment]
Looks good. Merging, thanks @SethosII

**Pull request merged**